apps/jobs/utils.py

```diff
@@ -15,9 +15,11 @@
 
 from .constants import submission_status_to_exclude
 from .models import Submission
+from challenges.models import ChallengePhaseSplit
```

Comment:
Can you please add this import statement below this line: https://github.com/Cloud-CV/EvalAI/pull/2798/files#diff-4678e5f644fe042c9dceeed05d57d66fL14